submissions/mustbefail/html-css-popup/style.css

```diff
@@ -0,0 +1,195 @@
+* {
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: #aaa;
+}
+
+a:focus {
+
+    border: 2px solid rgb(33, 180, 248, 0.5);
+
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+  
+}
+
+#menu {
+  margin-left: auto;
+  width: 400px;
+  padding: 20px;
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+}
+
+#menu > li {
+  display: inline-block;
+  width: 50px;
+  text-align: center;
+  margin-right: 20px;
+}
+
+#menu > li > a {
+  font-size: 20px;
+}
+
+#menu > li > a:hover {
+  color: rgb(105, 105, 105);
+}
+
+#menu .btn {
+  position: relative;
+}
+
+#menu input {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  opacity: 0;
+  bottom: 0;
+  left: 0;
+  margin: 0;
+  z-index: 3;
+  cursor: pointer;
+}
+
+#menu .menu {
+  position: relative;
+}
+
+#menu input:checked ~ .container {
+  display: block;
+}
+
+.container {
+  position: absolute;
+  top: 55px;
+  right: -100px;
+  transition: display 0.2s;
+  display: none;
+}
+
+.wraper {
+  position: relative;
+  overflow: auto;
+  max-height: 500px;
+  transition: height 0.3s;
+  padding: 0;
+  margin: 0 auto;
+  width: 320px;
+  background: white;
+  border-radius: 5px;
+  box-shadow: 5px 6px 45px rgba(0,0,0,0.3);
+  border: 1px solid rgb(223, 223, 223);
+}
+
+.popup {
+  margin: 0;
+  padding: 25px 0;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-around;
+}
+
+.popup li{
+  list-style: none;
+  width: 80px;
+  height: 80px;
+  margin-bottom: 20px;
+}
+
+.popup li a {
+
+  box-sizing: border-box;
+  display: block;
+  padding: 5px;
+  text-align: center;
+  text-decoration: none;
+  color: #333;
+  
+}
+
+.popup li a:hover {
+  box-shadow: 1px 1px 23px inset rgba(75, 75, 75, 0.1);
+}
+
+.popup li a img{
+  width: 100%;
+  height: 50px;
+  margin-bottom: 5px;
+}
+
+
+#menu input.more-items {
+  position: absolute;
+  opacity: 0;
+  bottom: 0;
+  left: 0;
+  margin: 0;
+  z-index: 3;
+  cursor: pointer;
+  height: 48px;
+}
+
+input.more-items:checked{
+  display: none;
+}
+
+input.more-items:checked ~ .add-more{
+  display: none;
+}
+
+input.more-items:checked ~ #popup2{
+  display: flex;
+}
+
+.add-more a {
+  width: 100%;
+  height: 48px;
+  background: rgb(209, 209, 209);
+  text-align: center;
+  line-height: 3;
+  display: inline-block;
+  color: rgb(105, 105, 105);
+}
+
+.add-more a:hover {
+  box-shadow: 1px 1px 43px inset rgba(75, 75, 75, 0.2);
+}
+
+
+#popup2 {
```

Comment:
It's better to use some more consistent class names, not popup popup2 popup3....